include/grpcpp/server_impl.h

```diff
@@ -368,12 +370,15 @@ class Server : public grpc::ServerInterface, private grpc::GrpcLibraryCodegen {
   // Handler for callback generic service, if any
   std::unique_ptr<grpc::internal::MethodHandler> generic_handler_;
 
-  // callback_cq_ references the callbackable completion queue associated
+  // callback_cq_shards_references the callbackable completion queues associated
   // with this server (if any). It is set on the first call to CallbackCQ().
   // It is _not owned_ by the server; ownership belongs with its internal
   // shutdown callback tag (invoked when the CQ is fully shutdown).
   // It is protected by mu_
-  CompletionQueue* callback_cq_ = nullptr;
+  CompletionQueue* callback_cq_shards_ = nullptr;
```

Comment:
perhaps using inlined vector or even vector is preferred now that there is stdlib++ is allowed?

Reply:
stdlib++ is allowed?

...std::map<> is allowed? (Asking for other work)
I assume absl is currently not allowed, still, right?

Reply:
Sure, this can be a vec.

Reply:
I thought Esun did that a few days back.  Please confirm with Vijay and/or Mark though.

Reply:
libstdc++ is allowed now, but not yet absl. That said, you still have to use our allocator and not the stock one, according to what I just confirmed from @veblush

Reply:
Since I posted my comment, @markdroth added a note: "so for now, let's keep the restrictions that we had previously been using: use stuff from std:: only in iomgr and gprpp directories.  if there's something that we want to use that does allow overriding its allocator, then we can define our own version in the gprpp directory with the overridden allocator, and then use that version in the rest of our code."

Reply:
I guess for now I can stick to InlinedVector.

Reply:
Actually - cannot directly use std::vec or InlinedVector for the CQs themselves since we cannot tie the memory lifecycle of the completion queue to the server due to the current ownership semantics. Instead: will use a std::vector for the array of pointers to sharded completion queues.